features/export.feature

```diff
@@ -412,3 +412,33 @@ Feature: cli-kintone export command
     When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --guest-space-id 9999999999999999999"
     Then I should get the exit code is non-zero
     And The output error message should match with the pattern: "\[403] \[CB_NO02] No privilege to proceed"
+
+  Scenario: CliKintoneTest-124 Should return the record contents successfully, including table data, when exporting the records
+    Given Load app ID of the app "app_for_export_table" as env var: "APP_ID"
+    And Load app token of the app "app_for_export_table" with exact permissions "view" as env var: "API_TOKEN"
+    And The app "app_for_export_table" has no records
+    And The app "app_for_export_table" with table field have some records as below:
+      | * | Text_0 | Text          | Number |
+      | * | Lisa   | Lisa Pink     | 10     |
+      | * | Rose   | Rose Roseanne | 20     |
+      | * | Bob    | Bob Dylan     | 30     |
+      | * | Jenny  | Jenny_1       | 40     |
+      |   | Jenny  | Jenny_2       | 50     |
+    When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN"
+    Then I should get the exit code is zero
+    And The app "app_for_export_table" with table field should has records as below:
+      | * | Text_0 | Table | (Table.Text)  | (Table.Number) |
+      | * | Lisa   | \d+   | Lisa Pink     | 10             |
+      | * | Rose   | \d+   | Rose Roseanne | 20             |
+      | * | Bob    | \d+   | Bob Dylan     | 30             |
+      | * | Jenny  | \d+   | Jenny_1       | 40             |
+      |   | Jenny  | \d+   | Jenny_2       | 50             |
+
+  Scenario: CliKintoneTest-125 Should return only field codes when exporting the records of the app which no data
+    Given Load app ID of the app "app_for_export_table" as env var: "APP_ID"
+    And Load app token of the app "app_for_export_table" with exact permissions "view" as env var: "API_TOKEN"
+    And The app "app_for_export_table" has no records
+    When I run the command with args "record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN"
+    Then I should get the exit code is zero
+    And The app "app_for_export_table" with table field should has records as below:
```

Comment:
@tuanphamcybozu 

If we use this step, It does not verify the header row.
The header row will be skipped.
Please check it again.
Thank you.